### ISSUE TITLE
Output changes when inference trained PEFT model, what's missing?

### ISSUE THREAD

**Issue by ruian0** [huggingface/peft#290]
I can save models from specific checkpoint following the discussion in #273. Thank you for the help!

For inference, I created an `inference.py` file in which I have 
```
peft_directory = ... # /path/to/checkpoint/adapter_model

peft_config = PeftConfig.from_pretrained(peft_directory)
model = LlamaForSequenceClassification.from_pretrained(peft_config.base_model_name_or_path)
model = PeftModel.from_pretrained(model, model_id=peft_directory)
model.eval()

text = "this is a test"
inputs = tokenizer(text, truncation=True, padding="max_length", max_length=max_sql_len, return_tensors="pt")

logits = model(**inputs).logits
```

However, the values of logits still change every time I run the cmd of `python inference.py` with the same input.

Can anyone provide more information for the next step of debugging? Thanks!

**Comment by ruian0** [comment]
not related to peft, closing
